docs/workforce-management-framework/topics/pro-working-with-templates-on-rhmap.adoc

```diff
@@ -0,0 +1,68 @@
+[id='{context}-pro-working-with-templates-on-rhmap']
+= Working with {Feature} Templates in {ProductShortName}
+
+This section includes:
+
+. xref:{context}-creating-a-new-wfm-project-in-rhmap[Creating a new {Feature} Project in {ProductShortName}]
+. xref:{context}-setting-up-and-deploying-the-wfm-demo-cloud-app[Setting Up and Deploying the WFM Demo Cloud App]
+. xref:{context}-setting-up-and-deploying-the-wfm-demo-portal-app[Setting Up and Deploying the WFM Demo Portal App]
+. xref:{context}-setting-up-and-building-the-wfm-demo-mobile-app[Setting Up and Building the WFM Demo Mobile App]
+
+[discrete]
+== Prerequisites
+
+Ensure you have:
+
+* Access to the {ProductShortName} Studio
+
+[id='{context}-creating-a-new-wfm-project-in-rhmap']
+[discrete]
+== Creating a new {Feature} Project in {ProductShortName}
+
+. Log on to the {ProductShortName} Studio
+. On the top navigation bar, select *Projects*
+. Select *New Project*
+. Look for the _WFM Project_ and click on *Choose*
+. Give the _WFM Project_ a name and click on *Create*
+.. You can rename the individual applications within the _WFM Project_
+. Once the _WFM Project_ is created, click *Finish*
+.. You are brought to the _WFM Project_ homepage
+
+[id='{context}-setting-up-and-deploying-the-wfm-demo-cloud-app']
+[discrete]
+== Setting Up and Deploying the WFM Demo Cloud App
+
+. On the _WFM Project_ homepage, click on the *WFM Demo Cloud App*
+. In the left side navigation panel, click on *Data Browser*
+. If the database requires an update, select *Upgrade database*
+.. A warning message appears, click on *Upgrade Now* to proceed
+. Once the upgrade is complete, click *Next*
+. Navigate to the deployment screen by clicking *Deploy*
+. On the deployment screen, select *Node.js - 6.1.9* from the *Choose a Runtime* list
```

Comment:
There is no runtime 6.1.9 on zeta, only 6.9.1. I hope that I was not testing on an old version as I asked for the latest representation of the application "I am wanting to test https://github.com/feedhenry-raincatcher/raincatcher-docs/pull/72/files against the latest representation of the application"

Reply:
good catch

Reply:
As from email we should use Node 6 LTS  and this should be variable.

Reply:
I'll update this and apply in a general review of repoo